tests/scripts/branch-history

```diff
@@ -149,7 +149,7 @@ foreach my $sha (@commits) {
         # File was added
         $blob_size += int(`git cat-file -s $dst`); }
       elsif ($mode eq "DD") { }
-      elsif ($mode eq "MM") {
+      elsif ($mode eq "MM" || $mode eq "MR") {
```

Comment:
I was getting the error "Unknown git status letter" on a different branch without this change